src/app/projects/projects-list/projects-list.component.html

```diff
@@ -97,7 +97,7 @@
         <td class="vf-table__cell">{{ project.date }}</td>
         <td class="vf-table__cell">
           <p style="margin-top: 0">{{ project.title }}</p>
-          <app-author-names [authors]="project.authors"></app-author-names>
+          <app-author-names *ngIf="project.authors.length" [authors]="project.authors"></app-author-names>
```

Comment:
What happens if the `authors` array is empty? We would still show the publications, right?
Is that OK to show the publications without authors? Just a question, I don't know the correct answer.

Reply:
Yes, it's okay. Discussed with @clairerye

Reply:
If we do have a publication but no authors, something has probably gone wrong though